scripts/guards/block-dangerous-git.sh

```
@@ -0,0 +1,203 @@
+if command -v jq >/dev/null 2>&1; then
+  cmd=$(printf '%s' "$input" | jq -r '.tool_input.command // empty')
+else
+  cmd=$(printf '%s' "$input" \
+    | sed -n 's/.*"command"[[:space:]]*:[[:space:]]*"\(.*\)".*/\1/p')
+fi
+[ -z "$cmd" ] && exit 0
+
+# What we match on: escaped quotes then quoted and backtick-wrapped regions
+# stripped out (a trigger phrase in a markdown code span of a gh body).
+scan=$(printf '%s' "$cmd" \
+  | sed -e 's/\\"//g' -e 's/"[^"]*"//g' -e "s/'[^']*'//g" -e 's/`[^`]*`//g')
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)